cardano-db/src/Cardano/Db/Operations/Delete.hs

```diff
@@ -102,106 +131,230 @@ deleteBlocksBlockId trce txOutTableType blockId = do
       CMinIdsWrapper (MinIds m1 m2 m3) -> isJust m1 && isJust m2 && isJust m3
       VMinIdsWrapper (MinIds m1 m2 m3) -> isJust m1 && isJust m2 && isJust m3
 
--- (MinIds m1 m2 m3) isJust m1 && isJust m2 && isJust m3
+mkRollbackSummary :: [(Text, Int64)] -> Text
+mkRollbackSummary logs = "\n ----------------------- Rollback Summary: ----------------------- \n" <> formattedLog
```

Comment:
Could this become a bit less verbose? For example we could split out logs with 0 deletes and print them in a single row
Count 0: TableA, TableB ...

Reply:
updated logs see PR description for updated screenshot